cg/services/upload_service/upload_config.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+
+from cg.constants.priority import SlurmAccount
+
+
+class UploadConfig(BaseModel):
+    email: str
+    log_dir: str
+    account: SlurmAccount
```

Comment:
Which email and log directory should we use? Same as in data delivery?

Reply:
Sounds reasonable to me to use the same, but I don't know much about what the values represent...

Reply:
This should go to `clinical-logwatch` source: @karlnyr

Reply:
Great! Thanks